recipe/meta.yaml

```diff
@@ -82,3 +83,4 @@ extra:
     - patricksnape
     - pelson
     - scopatz
+    - mbargull
```

Comment:
I suppose it makes sense to add me; but we also have a long-ish list of maintainers here, so adding me isn't really necessary -- I don't mind either way.